hr_payslip_ytd_amount/hr_payslip.py

```diff
@@ -31,6 +31,22 @@ def compute_sheet(self, cr, uid, ids, context=None):
         )
         self.compute_lines_ytd(cr, uid, ids, context=context)
 
+    def _search_employee_payslips_ytd(self, cr, uid, ids, context=None):
```

Comment:
Could you update the code with the new api, it's a small amount so it will not take that much